tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml

```diff
@@ -1,3 +1,21 @@
+trigger:
+  branches:
+    include:
+    - main
```

Comment:
I think what needs to be done is modifying the other pull request pipelines' yaml file so that they won't get auto-triggered by js changes, instead of modifying this one.

Reply:
I am testing if Web CI will be triggered when none-web file is changed.

Reply:
But I think we need to merge the yml changes to main before it would work?

Reply:
Even only none-web files is changed, we still need to run web CI pipeline. Because web packages depend on the core runtime.
In general, if the pipeline could still work normally if we delete some of the source code folders at the very beginning of the pipeline, it means these folders would not impact this pipeline such we can avoid triggering this pipeline if a PR only changes the deleted folders.